Create a general pull_request_template.md (#703)
devicons/devicon: .github/drafts/pull_request_template.md

---
name: Pull Request Template
about: Add a new icon or feature to the repo.
title: 'New Icon/Feature: [NAME]'
assignees: ''

---

> # New Icon Section

Reviewer: ```suggestion
# New Icon Section

```

> If you are adding a new icon, delete the **New Feature** section down below and follow the steps listed here.

Reviewer: ```suggestion
If you are adding a new icon, delete the **New Feature** section below, and follow the steps listed there.
```

> ## Double check these details before you open a PR**

Reviewer: ```suggestion
## Double check these details before you open a PR**

```

> *Tick the checkboxes to ensure you've done everything*

Reviewer: ```suggestion
*Tick the checkboxes to ensure you've done everything correctly*
```

- [] PR does not match another non-stale PR currently opened
- [] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)* as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)

> - [] Your icons are put in a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)

Reviewer: ```suggestion
- [] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
```

- [] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

> Refer to the [`CONTRIBUTING.md`](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#contributing-to-devicon) for more details.

Reviewer: ```suggestion
Refer to the [contributing](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#contributing-to-devicon) guidelines for more details.
```

> ## Link to the official page to prove your SVG is correct and up to date.

Reviewer: ```suggestion
## Link to the official page to prove your SVG is correct and up-to-date.
```

> *Link goes here*

Reviewer: This will make it a comment and only visible when making the PR or editing the comment.
```suggestion
<!-- Link goes here -->
```

> **Please link an issue that this PR should solve if it's applicable**

Reviewer: ```suggestion
<!-- Please link an issue that this PR should solve if it's applicable -->
```

> 

Reviewer: ```suggestion
```

> # New Feature Section

Reviewer: ```suggestion
# New Feature Section

```

> If you are adding a new icon, delete the **New Icon** section up above and follow the steps listed here.

Reviewer: ```suggestion
If you are adding a new icon, delete the **New Icon** section up above, and follow the steps listed there.
```

> ## Double check these details before you open a PR**

Reviewer: ```suggestion
## Double check these details before you open a PR**

```

> *Tick the checkboxes to ensure you've done everything*

Reviewer: ```suggestion
*Tick the checkboxes to ensure you've done everything correctly*
```

- [] PR does not match another non-stale PR currently opened
- [] PR name matches the format *New Feature: brief description of feature*

## This PR adds/fixes...

*List your features here and the benefits they bring.*

## Notes

> *List anything note-worthy here (potential issues, this needs to be merged to `master` before working, etc....).*
> *Don't forget to link any issues that this PR will solved.*

Reviewer: ```suggestion
<!-- List anything note-worthy here (potential issues, this needs to be merged to `master` before working, etc....).
Don't forget to link any issues that this PR will solve. -->
```

> **Please link any issue that this PR should solve if it's applicable**

Reviewer: ```suggestion
<!-- Please link any issue that this PR solves if applicable -->
```
